charts/kube-prometheus-stack/templates/alertmanager/service.yaml

```diff
@@ -40,6 +40,9 @@ spec:
       port: {{ .Values.alertmanager.service.port }}
       targetPort: {{ .Values.alertmanager.service.targetPort }}
       protocol: TCP
+{{- if .Values.alertmanager.service.additionalPorts }}
+{{ toYaml .Values.alertmanager.service.additionalPorts | indent 2 }}
```

Comment:
It would be nice to add this option in the values.yaml file here https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L272

Just to let users know that this option is available.

Reply:
Thanks. Just added that.